nncf/common/debug.py

```diff
@@ -0,0 +1,36 @@
+"""
```

Comment:
I would suggest moving `nncf/common/debug.py` to `nncf/common/utils/debug.py`

Reply:
Done